docs/start/quickstart.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Quickstart
+position: 2
+---
+# Quickstart
+
+Infra is rolling out it's Software as a Service offering which makes it easier than ever to setup infrastructure access for your organization. This page will show you everything you need to do to get started.
+
+As you scroll down, you will see a lot of text and might think that this is going to be hard. But really there are 5 steps as shown in the Table of Contents to the right. Everything else is just describing what to do just in case you run into any issues.
+
+## Signup to Create a New Organization
+
+{% callout type="info" %}
+
+We are slowly rolling out our SaaS product. If you are interested in onboarding before we open it up completely, let us know. Until then you can [install the self-hosted version of the Infra Server](../reference/selfhosted.md).
```

Comment:
Great callout! Let's link this to a signup form once that's ready vs sending to self-hosted

Reply:
send me the signup form and it will be done